src/layers/domain/core/device/v1.py

```diff
@@ -20,6 +20,8 @@
 
 UPDATED_ON = "updated_on"
 DEVICE_UPDATED_ON = f"device_{UPDATED_ON}"
+MHS_DEVICE_NAME = "Product-MHS"
+AS_DEVICE_NAME = "Product-AS"
```

Comment:
These have been removed in the current release and added to src/layers/sds/epr/constants.py for consistency with the etl :) 
https://github.com/NHSDigital/connecting-party-manager/pull/428/files#diff-3fce374cb1f879f10c43b6153cfe9635be0c12ff2ec1e43797f13f11d3fe688bR1